spec/spec_helper.rb

```diff
@@ -1 +1,2 @@
 require 'beaker/puppet_install_helper'
+require 'beaker/ca_cert_helper'
```

Comment:
Do you think this should be require'd from the lib/beaker/puppet_install_helper.rb so that users don't have to add an extra require like this?